### DISCUSSION TITLE
Ability to blacklist/remove/ban specific words

### DISCUSSION THREAD

**Discussion by Kelvets** [monkeytypegame/monkeytype#2334] [Ideas]
I do tests with the English 5k setting and every time the word "sovereignty" comes up it trips me up. Since I realistically never ever type this word in my daily life, I'd like to remove it. 
But even worse is the word "extraordinary". That's because I have a macro running at all times in my computer that substitutes "ao" with "ão", since the latter is an extremely more common combination in Portuguese (my mother language) than the former, and the ~ key is far away on the keyboard, so I save myself one awkward keystroke every time. But if I forget to disable the macro, it comes out as "extrãordinary" and I get an instant error on the A every time that I can't correct despite trying several times. By the time I remember what's happening, I've already lost so much time that it basically invalidated my whole run.

I'd really appreciate if Monkeytype had a list where you could add "banned" words, which would never show up in the tests.

**Comment by kiiatto** [comment]
i know i'm about three or almost four years late but this feature still doesn't exist as far as i'm aware, and i get tripped up typing "follow" all the time on QWERTY because i have to use my right ring for both O's and L's in that word and it takes ages to type compared to like everything else sobbbbb

**Comment by Leonabcd123** [comment]
You can enter the words you don't want in the "exclude" input box in the words filter modal (`custom` -> `change` -> `words filter` -> enter blacklisted words in `exclude` input -> `set`).
